src/OMSimulatorLib/SystemTLM.h

```diff
@@ -57,6 +58,10 @@ namespace oms3
     SystemTLM& operator=(SystemTLM const& copy); ///< not implemented
 
   private:
+    void* model;
+    std::string address;
+    int managerPort;
+
     // simulation information
     // ip
     // managerport
```

Comment:
The comments can be removed if implemented.

Reply:
Done.